### DISCUSSION TITLE
SEO The GitHub Page

### DISCUSSION THREAD

**Discussion by ddxv** [janreges/siteone-crawler#29] [Ideas]
@janreges Love the product! Recently reformatted my computer but then couldn't remember the name of your product. I tried searching "seo" on GH and SiteOne doesn't show up in the first few pages.

Looking at the main page, "SEO" only shows up once in the keywords and once in the README.md.

Perhaps that's because SEO isn't the main point of this project, but just happens to be what I was looking for when I was Googling/Redditing/GitHubbing looking for your tool. Perhaps you could try adding some keywords for searching to the

Title
 Short Description
 Long Description -> Add a few more
 
 Anyways, just a thought. Very glad I found it again. Thanks!

**Comment by janreges** [comment]
Hi, @ddxv,

I'm very glad you like the tool and find it useful. I will be very happy if you share it with your colleagues or friends ;-)

Thank you also for your feedback with the inappropriate description. I have modified the GitHub project description to be more searchable with such keywords. I will try to find time in the next days to optimize other important texts as well.

**Reply by ddxv** [reply]
Awesome 👍!

**Comment by a-matthew** [comment]
Hi, I didn't want to start a new topic just to say that I want to thank you for creating this tool.

It has helped me personally. It's incredibly convenient that you also support an AppImage for Linux systems - it makes the installation process streamlined and lightweight.

If that's anyhow useful - I learned about this crawler by sheer luck from the posts on Reddit and on Hacker News.
From my non-professional perspective - I find the UI and overall experience excellent so far.
